### PR TITLE
Graceful shutdown fixes

### PR DESCRIPTION
**Pull request by maaktweluit**
Small fixes for the reported errors by @Krigpl :
```
  File "/home/admin_imapp/golem_repos/golem/golem/task/taskmanager.py", line 751, in get_progresses
    t.short_extra_data_repr(2200.0)
  File "/home/admin_imapp/golem_repos/golem/apps/rendering/task/renderingtask.py", line 226, in short_extra_data_repr
    "outfilebasename: {outfilebasename}, scene_file: {scene_file}".format(**l)
builtins.TypeError: format() argument after ** must be a mapping, not float
```
Added default of `None` and a query for data inside `short_extra_data_repr()`.
This also relates to #2460

```
  File "/home/admin_imapp/golem_repos/golem/golem/transactions/ethereum/ethereumtransactionsystem.py", line 79, in stop
    super().stop()
  File "/home/admin_imapp/golem_repos/golem/golem/core/service.py", line 62, in stop
    raise RuntimeError("service not started")
builtins.RuntimeError: service not started
```
Turned out `node.quit()` was calling `client.quit()` twice, removed one of them.

### CODE REVIEW

File: golem/task/taskmanager.py [hunk 1]
```diff
@@ -748,7 +748,7 @@ def get_progresses(self):
                     t.get_total_tasks(),
                     t.get_active_tasks(),
                     t.get_progress(),
-                    t.short_extra_data_repr(2200.0)
```
Comment (Krigpl) [inline golem/task/taskmanager.py:751]: What was that number and why it's gone ?

Comment (maaktweluit) [reply golem/task/taskmanager.py:751]: i think it is a `perf_index` value, used by all `query_extra_data` functions.
In this case it is strange since this function expects a `extra_data` dict object.
Looking back at the history this was added 11 months ago by Jacek or inext. with a comment to query the data inside.

i have followed the comment/fixme and moved the query inside the blenderrender class.
We can use the FIXME issue to continue this fix for the other renderers

File: golem/node.py [hunk 2]
```diff
@@ -30,10 +30,10 @@
 logger = logging.getLogger(__name__)
 
 
-class ShutdownResponse(Enum):
-    quit = "quit"
-    off = "off"
-    on = "on"
+class ShutdownResponse(IntEnum):
```
Comment (Krigpl) [inline golem/node.py:33]: Why was that change needed ?

**Review by Krigpl — APPROVED** [review]
thumbupglitch

File: golem/node.py (hunk 2 above)
Comment (maaktweluit) [reply golem/node.py:33]: Twisted is not able to serialise `Enum`, it does for `IntEnum`

**Review by mfranciszkiewicz — APPROVED** [review]
